### PR TITLE
update text generation docs.

### PR DESCRIPTION
**Pull request by Vaibhavs10**
[DRAFT] - Feedback/ comments very much appreciated!

### CODE REVIEW

**Review by Wauplin — APPROVED** [review]
Nice update!

**Comment by Vaibhavs10** [comment]
cc: @osanseviero @pcuenca thoughts?

File: packages/tasks/src/tasks/text-generation/about.md
Comment (pcuenca) [inline packages/tasks/src/tasks/text-generation/about.md:47]:
```suggestion
There are language models that can input both text and image and output text, called vision language models. [IDEFICS 2](https://huggingface.co/HuggingFaceM4/idefics2-8b) and [MiniCPM Llama3 V](https://huggingface.co/openbmb/MiniCPM-Llama3-V-2_5) are good examples. They accept the same generation parameters as other language models. However, since they also take images as input, you have to use them with the `image-to-text` pipeline. You can find more information about this in the [image-to-text task page](https://huggingface.co/tasks/image-to-text).
```

Also maybe PaliGemma?

File: packages/tasks/src/tasks/text-generation/about.md
Comment (pcuenca) [inline packages/tasks/src/tasks/text-generation/about.md:7]:
```suggestion
A model trained for text generation can be later adapted to follow instructions. You can try some of the most powerful instruction-tuned open access models like Mixtral 8x7B, Cohere Command R+, Meta Llama3 70B, etc [at Hugging Chat](https://huggingface.co/chat).
```

**Review by pcuenca — APPROVED** [review]
Thanks for updating!

**Comment by Vaibhavs10** [comment]
Thanks for the suggestions! ❤️

File: packages/tasks/src/tasks/text-generation/about.md
Comment (osanseviero) [inline packages/tasks/src/tasks/text-generation/about.md:7]:
```suggestion
A model trained for text generation can be later adapted to follow instructions. You can try some of the most powerful instruction-tuned open-access models like Mixtral 8x7B, Cohere Command R+, and Meta Llama3 70B [at Hugging Chat](https://huggingface.co/chat).
```

Comment (osanseviero) [inline packages/tasks/src/tasks/text-generation/about.md:7]: I would suggest to not specify the models, as otherwise we'll have to maintain the list

File: packages/tasks/src/tasks/text-generation/about.md
Comment (osanseviero) [inline packages/tasks/src/tasks/text-generation/about.md:41]: Qwen links to CodeQwen Chat

Comment (osanseviero) [inline packages/tasks/src/tasks/text-generation/about.md:41]: I think some of these were RLHF-ed

File: packages/tasks/src/tasks/text-generation/about.md
Comment (osanseviero) [inline packages/tasks/src/tasks/text-generation/about.md:47]: LGTM, but should be moved to a new task page + the new pipeline once it's shipped cc @merveenoyan

Comment (Vaibhavs10) [reply packages/tasks/src/tasks/text-generation/about.md:47]: SGTM!

File: packages/tasks/src/tasks/text-generation/about.md
Comment (Vaibhavs10) [reply packages/tasks/src/tasks/text-generation/about.md]: IMO, it's good to add the model names, helps with SEO I think. I'm happy to take ownership of this once every month.

File: packages/tasks/src/tasks/text-generation/about.md
Comment (Vaibhavs10) [reply packages/tasks/src/tasks/text-generation/about.md:41]: Good catch! Fixing this.

Comment (Vaibhavs10) [reply packages/tasks/src/tasks/text-generation/about.md]: Fixed in the latest commit.

**Comment by Vaibhavs10** [comment]
Will wait for @osanseviero and @lewtun's review/ LGTM before merging.

**Review by osanseviero — APPROVED** [review]
LGTM!

**Pull request merged**
